migrations/0034-add-userSocialMedia-table.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const TABLE_NAME = 'UserSocialMedias';
```

Comment:
nit: grammar with plurals (I just noticed this in this PR, let's have this table name be "UserSocialMedia" instead of "UserSocialMedias"
```suggestion
const TABLE_NAME = 'UserSocialMedia';
```